src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -370,16 +393,26 @@ export class OsdListComponent extends ListWithDetails implements OnInit {
     }
   }
 
+  filterIndivFlags(flags: string[]) {
+    return flags.filter((f) => this.indivFlagNames.includes(f));
+  }
+
+  filterClusterWideFlags(flags: string[]) {
+    return flags.filter((f) => !this.disabledFlags.includes(f));
+  }
```

Comment:
This is causing too many change detections.
We should calculate this when create the osds list.
```suggestion
```